README.md

```diff
@@ -71,7 +71,7 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+The code would make most sense as a MODEL method, as it would be a proper separation of concerns. The controller is the coordinator that takes parameters and returns HTML, but it should not be making business logic decisions such as determining the average rating of a song. We would create an avq_rating method in the song model that can then be called in the SongsController and display the HTML/CSS for the appropriate color.
```

Comment:
I could totally see why you might say Model method. We were looking for a helper method. Here is the answer:

```
Helper method, because these are intended to "help" render HTML based on conditions. Controller methods concern how data moves between routes. Model mehtods concern how data goes to and comes from the database.
```